src/components/menu/menu.tsx

```diff
@@ -42,14 +41,11 @@ export const DesktopMenu = () => {
           <p className={classNames(styles.menuItemText, globalStyles.textSmall)}>History</p>
         </div>
       </NavLink>
-      <NavLink activeClassName={styles.menuActiveItem} to="/policies">
-        <div className={styles.menuItem}>
-          <div className={styles.menuActiveLine} />
-          <PoliciesIcon aria-hidden />
-          <p className={classNames(styles.menuItemText, globalStyles.textSmall)}>My Policies</p>
-        </div>
-      </NavLink>
-      <NavLink activeClassName={styles.menuActiveItem} to="/claims">
+      <NavLink
+        activeClassName={styles.menuActiveItem}
+        to="/claims"
+        isActive={() => pathname.startsWith('/claims') || pathname.startsWith('/policies')}
```

Comment:
Why also `/policies`? For legacy reasons?

Reply:
So that it highlights the My Claims nav item as active when we are on the Policy Details and New Claim pages (`/policies/<ID>`,  `/policies/<ID>/claims/new`), now that we don't have the My Policies nav item anymore.